pkg/util/taints/taints_test.go

```diff
@@ -542,42 +692,87 @@ func TestParseTaints(t *testing.T) {
 		expectedErr            bool
 	}{
 		{
-			name:        "invalid spec format",
+			name:        "invalid empty spec format",
 			spec:        []string{""},
 			expectedErr: true,
 		},
 		{
-			name:        "invalid spec format",
+			name:        "invalid spec format with a special char",
```

Comment:
which special char?

Reply:
It would be more appropriate to change it to `invalid spec format without effect`.